src/render/Annotation.cpp

```diff
@@ -10,6 +10,7 @@
 #include <QColor>
 #include <QImage>
 #include <QPainter>
+#include <QPainterPath>
```

Comment:
Why does adding this include fix this?

The deprecation warning indicates that use of `fm.width` should be changed?

Reply:
Sorry, copied the wrong error message.

```
/home/jduchniewicz/Packages/displaz/src/render/Annotation.cpp:36:18: error: aggregate ‘QPainterPath path’ has incomplete type and cannot be defined
   36 |     QPainterPath path;
      |                  ^~~~
```

Reply:
Oh I see. In that case this change makes perfect sense, thanks!